packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-swift-visitor.test.ts

```diff
@@ -13,12 +13,13 @@ const getVisitor = (
   selectedType?: string,
   generate: CodeGenGenerateEnum = CodeGenGenerateEnum.code,
   isTimestampFieldsAdded: boolean = true,
+  emitAuthProvider: boolean = true,
```

Comment:
CLI PR to add new feature flag: https://github.com/aws-amplify/amplify-cli/pull/7447